palette/src/lib.rs

```diff
@@ -313,8 +313,9 @@ pub use oklab::{Oklab, Oklaba};
 pub use oklch::{Oklch, Oklcha};
 #[doc(inline)]
 pub use rgb::{
-    AdobeRgb, AdobeRgba, GammaSrgb, GammaSrgba, LinAdobeRgb, LinAdobeRgba, LinRec2020, LinSrgb,
-    LinSrgba, Rec2020, Rec709, Srgb, Srgba,
+    AdobeRgb, AdobeRgba, DciP3, DciP3Plus, DisplayP3, GammaSrgb, GammaSrgba, LinAdobeRgb,
+    LinAdobeRgba, LinDciP3, LinDciP3Plus, LinDisplayP3, LinRec2020, LinSrgb, LinSrgba, Rec2020,
+    Rec709, Srgb, Srgba,
```

Comment:
This is getting crowded. I can scale it back later if it messes with the other PR, but there's no need to export everything to the top module. I should have thought of it earlier.